tests/test_snr.py

```diff
@@ -117,6 +117,13 @@ def test_DetectorStates(data_parameters, writer):
     ]
     assert compare_multi_detector_states_series(*mds)
 
+    # test again with plain list instead of np.array
+    mds2 = pyfstat.DetectorStates().get_multi_detector_states(
+        Tsft=writer.Tsft, timestamps={key: list(common_ts) for key in ("H1", "L1")}
+    )
+    assert compare_multi_detector_states_series(*[mds[0], mds2])
```

Comment:
```suggestion
    assert compare_multi_detector_states_series(mds[0], mds2)
```
No unpacking is needed here